From 8f546d337c41af152ec5941c2a7c0b0b187be2c9 Mon Sep 17 00:00:00 2001
From: "Mishra, ManishX" <manishx.mishra@intel.com>
Date: Thu, 21 Oct 2021 14:21:37 +0530
Subject: [PATCH] Revert "Adding a min_delay & vendor property to extract
 host_ip address"

This reverts commit e39dd72ca774538caf09cdf1952493c9a86cf26d.
---
 iio-client.cpp |  4 ++--
 sensor_hal.cpp | 18 +++++++++---------
 2 files changed, 11 insertions(+), 11 deletions(-)

diff --git a/iio-client.cpp b/iio-client.cpp
index 2387fab..7e95127 100644
--- a/iio-client.cpp
+++ b/iio-client.cpp
@@ -62,7 +62,7 @@ int iioClient::init(void)
 
     ctx = NULL;
     sensorCount = 0;
-    property_get("vendor.intel.ipaddr", value, " ");
+    property_get("ipaddr", value, " ");
     ctx = iio_create_network_context(value);
     if (!ctx) {
         ALOGE("Sensor: Error in Initializing IIO Client with N/W backend\n");
@@ -188,7 +188,7 @@ int iioClient::getPollData(sensors_event_t* data)
         data[k].sensor = iM[index].id;
         data[k].type = iM[index].type;
         data[k].version = sensorList[i].version;
-        data[k].timestamp = get_timestamp(CLOCK_BOOTTIME);
+        data[k].timestamp = get_timestamp(CLOCK_MONOTONIC);
         if (data[k].type == SENSOR_TYPE_ACCELEROMETER) {
            nb_channels = nb_channels - 1;
         }
diff --git a/sensor_hal.cpp b/sensor_hal.cpp
index bc48683..5933975 100644
--- a/sensor_hal.cpp
+++ b/sensor_hal.cpp
@@ -46,7 +46,7 @@ static const struct sensor_t sSensorList[MAX_SENSOR] = {
      1000,
      1.52e-5,
      0.0,
-     2000,
+     0,
      0,
      0,
      "android.sensor.accelerometer",
@@ -63,7 +63,7 @@ static const struct sensor_t sSensorList[MAX_SENSOR] = {
      1000,
      0.1,
      0.0,
-     2000,
+     0,
      0,
      0,
      "android.sensor.inclinometer",
@@ -80,7 +80,7 @@ static const struct sensor_t sSensorList[MAX_SENSOR] = {
      1000,
      0.1,
      0.0,
-     2000,
+     0,
      0,
      0,
      "android.sensor.gravity",
@@ -97,7 +97,7 @@ static const struct sensor_t sSensorList[MAX_SENSOR] = {
      1000,
      0.1,
      0.0,
-     2000,
+     0,
      0,
      0,
      "android.sensor.dev_rotation",
@@ -114,7 +114,7 @@ static const struct sensor_t sSensorList[MAX_SENSOR] = {
      1000,
      0.1,
      0.0,
-     2000,
+     0,
      0,
      0,
      "android.sensor.magn_3d",
@@ -131,7 +131,7 @@ static const struct sensor_t sSensorList[MAX_SENSOR] = {
      100,
      0.1,
      0.0,
-     2000,
+     0,
      0,
      0,
      "android.sensor.geomagnetic_orientation",
@@ -148,7 +148,7 @@ static const struct sensor_t sSensorList[MAX_SENSOR] = {
      100,
      0.1,
      0.0,
-     2000,
+     0,
      0,
      0,
      "android.sensor.relative_orientation",
@@ -165,7 +165,7 @@ static const struct sensor_t sSensorList[MAX_SENSOR] = {
      100,
      0.1,
      0.0,
-     2000,
+     0,
      0,
      0,
      "android.sensor.gyro_3d",
@@ -182,7 +182,7 @@ static const struct sensor_t sSensorList[MAX_SENSOR] = {
      100,
      0.1,
      0.0,
-     2000,
+     0,
      0,
      0,
      "android.sensor.als",
-- 
2.33.1

